### DISCUSSION TITLE
Dagger backend integration

### DISCUSSION THREAD

**Discussion by grouville** [dagger/dagger#867] [Help]
Let's say that, using Dagger, someone develops an awesome package and wants to integrate it in his SAAS. What is the current way to do it ?

For example: a user clicks on a button (on a website) -> dagger runs with a specific package -> outputs are being sent back to the server, etc ...

Is it possible atm ? If yes, how do we integrate it ? If not, is it something that will be on our roadmap anytime soon (or not) ?

**Comment by TomChv** [comment]
If I'm right, @Zardows has already link dagger with a backend, maybe he can explain how he did that ?

**Comment by routel-nolan** [comment]
Hi @grouville, can you explain what you got actually and what you want to do ?
